x-pack/plugins/ml/server/models/data_recognizer/modules/uptime_heartbeat/manifest.json

```diff
@@ -23,5 +23,6 @@
       "file": "datafeed_high_latency_by_geo.json",
       "job_id": "high_latency_by_geo"
     }
-  ]
+  ],
+  "tags": []
```

Comment:
I think this should be ["observability"]

Reply:
Updated in https://github.com/elastic/kibana/pull/168721/commits/ca79700393667389ca8e418f44990977db8817c3